### DISCUSSION TITLE
Need help to bring back the search (highlight?) bar at the bottom of the page

### DISCUSSION THREAD

**Discussion by iG8R** [hoothin/SearchJumper#102] [Q&A]
Hi.

Don't know how I did it the first and last time, but I somehow managed to bring up the search (highlight) bar at the bottom of the page to highlight all words that I input in this bar on the page. But now, тo matter how hard I looked, I couldn't find how it was done.
Please help me find how to bring up this search bar once again.

Here is a sketch of how it looked:
![image](https://github.com/hoothin/SearchJumper/assets/11407417/edf4b214-f5bb-4189-b5ca-9ade4d6c8911)

**Comment by hoothin** [comment]
+ shortcut
![image](https://github.com/hoothin/SearchJumper/assets/5361213/5340c6a2-4a22-4475-8ea5-df521fc0a2c1)

+ menu
![image](https://github.com/hoothin/SearchJumper/assets/5361213/18733b1a-6437-4ce2-9cdb-e6078d47f0d8)

**Reply by iG8R** [reply]
Oh, thank you very much!
How can I navigate through the found word/s on the page using the keyboard, i.e. find the next/previous occurrence of a word/words?

**Reply by hoothin** [reply]
+ right click/left click on the little number e.g. [1/9]
+ click navigate bar on the right
+ alt+click the highlight words on the page

**Reply by iG8R** [reply]
Thank you, but how could I do it with the keyboard? Ctrl+Up/Down or anything else?

**Reply by hoothin** [reply]
No way at now, but it can be made later.

**Reply by iG8R** [reply]
Thank you, you are great!

**Reply by hoothin** [reply]
You can switch highlight words with enter/shift+enter now. If you highlight multiple words, change them with left arrow/right arrow.
The shortcut effect only when search input is open.

**Reply by iG8R** [reply]
It's so convenient, thanks a lot!
